BUILD.md

```diff
@@ -59,7 +62,7 @@ in this repository for more details.
 
 ## Repository Content
 
-This repository contains the source code necessary to build the Vulkan
+This repository contains the source code necessary to build the Vulkan SC
```

Comment:
Should this read, "necessary to build the Vulkan and Vulkan SC validation layers"?

Reply:
sure. Done.